Simplified/NYPLSecrets.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+enum NYPLSecrets {
+  private static let salt: [UInt8] = [0]
+
+  static var feedbooks:String {
+    let encoded: [UInt8] = [0]
+    return decode(encoded, cipher: salt)
+  }
```

Comment:
There should be associated info here. Namely the profile url.

Reply:
Added